lib/Rest/VstResponse.cpp

```diff
@@ -144,8 +141,24 @@ void VstResponse::writeMessageHeader(VPackBuffer<uint8_t>& buffer) const {
         }
       }
     }
+  };
+  
+  std::string currentHeader;
+  builder.openObject();  // 4 == meta
+  for (auto& item : _headers) {
+    if (item.first.compare(0, StaticStrings::ContentTypeHeader.size(),
+                           StaticStrings::ContentTypeHeader) == 0) {
+      continue;
+    }
+    currentHeader.assign(item.first);
+    fixCase(currentHeader);
     builder.add(currentHeader, VPackValue(item.second));
   }
+  if (_contentType != ContentType::VPACK) { // fuerte uses VPack as default
```

Comment:
for content type "CUSTOM" this will write `Content-Type: `, because `rest::contentTypeToString(CUSTOM)` will return an empty string. Intentional?

Reply:
perhaps we need a more comprehensive fix

Reply:
I think we need to store a custom content type separately, or remove the special handling of the content-type altogether and handle it like all other headers.